src/service-registration/service-registration.controller.ts

```diff
@@ -0,0 +1,44 @@
+import {
+  Controller,
+  Get,
+  Post,
+  Body,
+  Delete,
+  Param,
+  Header,
+} from '@nestjs/common';
```

Comment:
das sieht komisch aus. ich würde es in eine zeile schreiben.
```suggestion
import { Controller, Get, Post, Body, Delete, Param, Header } from '@nestjs/common';
```